taskfiles/dev.yml

```diff
@@ -111,20 +111,6 @@ tasks:
     status:
       - test -f '{{.DOCKER_TAG_LIST_INSTALL_DIR}}/docker-tag-list'
 
-  install-goreleaser:
```

Comment:
Go releaser is still being used AFAIK. Why is this being removed?

Reply:
Oh, it's just re declared later in the same file. I get YAML unmarshaling problems for duplicate keys.. can't figure out why others don't..